pipeline_tools/pipelines/smartseq2/smartseq2.py

```diff
@@ -168,3 +177,19 @@ def _get_content_for_ss2_se_input_tsv(
     sample_id = metadata_utils.get_sample_id(primary_bundle)
     fastq_url = get_urls_to_files_for_ss2_se(primary_bundle)
     return fastq_url, sample_id
+
+
+references = {
+    # human
+    9606: {
```

Comment:
Not sure how things are done in python, but is there a way to avoid magic numbers here? E.g. create a named constant for the taxon IDs and use it everywhere `9606` is used.

Reply:
@samanehsan @rexwangcc any insight into the pythonic way?

Reply:
What I'd probably do here(and the above for optimus) is to:
- create another module (perhaps call it references.py) under `pipeline_tools/pipelines/smartseq2/references.py`
- define with Capitalized `REFERENCES` (PEP8: https://www.python.org/dev/peps/pep-0008/#constants)

when it comes to pythonic ways to avoid magic numbers here, I can think of 2 ways:

1. As @pshapiro4broad suggested, defined `HUMAN = 9606` at the top and use it as the dict keys. This is a bit risky since there are no concepts of "named constant" in Python world so the variables you defined can be overridden elsewhere.
2. Define all of the available  the `Enum` class:
```
>>> from enum import Enum
>>> class Reference(Enum):
... 	Human = 9606
... 	Mouse = 10080
```
and then get them by `Reference.Human.name` and `Reference.Human.value` accordingly. The `Enum` class is hashable and immutable so no one can reassign members (e.g. `Reference.Human = 'rex'` will raise some errors like "AttributeError: Cannot reassign members.")